multi_import/object_cache.py

```diff
@@ -32,37 +32,73 @@ def __init__(self, lookup_fields):
     def _default_factory(self):
         return defaultdict(set)
 
-    def _add_lookup_value(self, field, value, instance):
+    def is_multiple_fields(self, fields):
+        return (
+            hasattr(fields, '__iter__')
+            and not isinstance(fields, (str, bytes))
```

Comment:
I don't think this is wrong, but I'd argue that perhaps it makes more sense to EXPECT fields to be passed in as a certain type.  Like, list, or tuple.  I realize that this adds a sort of type expectation (hence the function is less flexible in terms of what is inputed), but the reason I prefer this because then it assures us that we aren't getting (and mistakenly allowing) garbage to enter.  There are two cases here that worry me in particular: 

1- Dictionaries! Do we want to allow the user to input dictionaries?  (i'll check other parts of the code to see if that could be handled or not, but right off the bat i'm thinking that we wouldn't be able to handle that)

2- What if in the future there's something else similar to `str` or `bytes` that is iterable but not in the way that we want it to be.  Like `unicode` in python 3.  I think limiting it just means that its less stuff we have to worry about later. 

Perhaps i'm missing some obvious reason why limiting to just a list or a tuple is bad or extremely undesireable, and if i am, please tell me :)